assignment/src/com/CodingQuestions/Question5/UserMainCode.java

```diff
@@ -0,0 +1,24 @@
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Set;
+
+class UserMainCode {
+
+    public static int getAverageOfOdd(HashMap<Integer,Integer> number1) {

```

Comment:
Pls use proper naming convention. You can use receivingMap instead of number1. Likewise for sending also